examples/01_prepare_data/mind_utils.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>same here, please move this to the utils and add docstrings. Also, it might be easier for our users to understand what this function does if the title is more explicit. Some ideas would be <code>load_glove_matrix</code>,  <code>generate_embeddings</code>,  <code>generate_embedding_matrix</code>  or any other you think is better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Also would be nice to have some comments in the codes what's going there. e.g. what's the data in l[0] vs l[1:] ? People who are not familiar w/ MIND dataset like me :-) will appreciate those comments.</p>

Reply:
<pre class="ql-syntax" spellcheck="false">wordvec = [float(x) for x in l[1:]]
if word in word_dict:
</pre><p>This part, seems we don't need to initialize wordvec if word is not in word_dict, meaning we can move wordvec = ... under if statement. That means, we can use 1 if-statement like:</p><p> </p><pre class="ql-syntax" spellcheck="false">if len(word) &gt; 0 and word in word_dict:
    do something here
</pre><p>or if we sure word_dict don't include any len(word) == 0, simply:</p><pre class="ql-syntax" spellcheck="false">if word in word_dict:
</pre><p> </p><p> </p>